lighthouse-core/scripts/collect-strings.js

```diff
@@ -0,0 +1,66 @@
+const fs = require('fs')
+const path = require('path')
+
+const LH_ROOT = path.join(__dirname, '../../')
+
+const strings = {}
+
+const bannedPathComponents = [
+  '/.git',
+  '/docs',
+  '/coverage',
+  '/scripts',
+  '/node_modules',
+  '/lighthouse-cli',
+  '/renderer',
+  '/test/',
+  '-test.js',
+  'lighthouse-viewer',
+  'lighthouse-extension',
+]
+
+function collectAllStringsInDir(dir) {
+  for (const name of fs.readdirSync(dir)) {
+    const fullPath = path.join(dir, name)
+    const relativePath = path.relative(LH_ROOT, fullPath)
+    if (bannedPathComponents.some(p => fullPath.includes(p))) continue
+
+    if (fs.statSync(fullPath).isDirectory()) {
+      collectAllStringsInDir(fullPath)
+    } else {
+      if (name.endsWith('.js')) {
+        console.log('Collecting from', relativePath)
+        const mod = require(fullPath)
+        if (!mod.UI_STRINGS) continue
+        for (const [key, value] of Object.entries(mod.UI_STRINGS)) {
+          strings[`${relativePath}!#${key}`] = value
```

Comment:
wdyt about just adopting the basic chrome i18n format here: 


```js
        "relativePathAndKey": {
          "message": "stringValue",
          // "description": "..."
        },
```

Like.. we _could_ translate again into that format, but perhaps its nicer to just start with it here.
also it looks like git diff doesnt handle small changes in these strings files well. considers the WHOLE block to be a change.

Reply:
yeah sg, done